flask_rebar/swagger_generation/marshmallow_to_swagger.py

```diff
@@ -674,76 +693,28 @@ def get_enum(self, obj, context):
         UUIDConverter(),
         ConstantConverter(),
     ]
+    if EnumConverter:
+        converters.append(EnumConverter())
```

Comment:
Instead just append it is the marshmallow type is not none for the enum converter